src/app/components/client/ExposureCard.tsx

```diff
@@ -35,84 +35,88 @@ type DetailsFoundProps = {
   icon: ReactElement;
 };
 
-export const ExposureCard = (props: Props) => {
+export const ExposureCard = (props: ExposureCardProps) => {
   const {
     exposureImg,
     exposureName,
     exposureType,
     exposureDetailsLink,
     dateFound,
-    statusPillContent,
     statusPillType,
   } = props;
 
   const [detailsOpen, setDetailsOpen] = useState(false);
+  const [isMobile, setIsMobile] = useState(false);
```

Comment:
Is it not possible to hide all the elements that listen to `isMobile` with media queries? That should be more performant AFAIK.

Otherwise, we have a `useMediaQuery` hook in Relay that might be useful to use in Monitor.